data/RecordAction.js

```diff
@@ -41,7 +70,7 @@ export class RecordAction {
      * @param {ActionCb} [c.actionFn] - called on store action activation.
      * @param {boolean} [c.disabled] - true to disable this item.
      * @param {boolean} [c.hidden] - true to hide this item.
-     * @param {PrepareFnCb} [c.prepareFn] - called prior to showing the action in the UI.
+     * @param {DisplayConfigFn} [c.displayConfigFn] - called prior to showing the action in the UI.
```

Comment:
Thinking we should use the name 'renderFn' or 'displayFn'?   This name is a bit of a mouthful, and I think of a 'config' as something you do once, when you construct an object...